script/DeployManagers.sol

```diff
@@ -130,6 +136,12 @@ contract DeployManagers is BaseDeployPeriphery {
       id: keccak256("VALIDATION_MANAGERS_REGISTRY"),
       contractAddress: validationManagersRegistry
     });
+    // Only for base we have to comment this, because global registry was deployed and deployer has no role to add it
+    /*config[5] = GlobalEarnRegistry.InitialConfig({
+      id: keccak256("MORPHO_REWARDS_MANAGER"),
+      contractAddress: morphoRewardsManager
+    });*/
+
```

Comment:
Actually, I think we can un-comment this. Since the contract won't be deployed (since it was deployed already)